croud/organizations/users/commands.py

```diff
@@ -31,19 +33,35 @@
 def org_users_add(args: Namespace):
     client = Client.from_args(args)
 
-    client.send(
+    data, errors = client.send(
         RequestMethod.POST,
         f"/api/v2/organizations/{args.org_id}/users/",
         body={"user": args.user, "role_fqn": args.role},
     )
-    client.print(keys=["user_id", "role_fqn", "organization_id"])
+    if data is not None and data["added"]:
```

Comment:
Guess it's ok to assume that the response contains the "data" key, however, to be on the safe, you could do:

```suggestion
    if data is not None and data.get("added", False):
```